src/main/java/seedu/address/model/internship/InternshipApplication.java

```diff
@@ -46,15 +46,15 @@ public Role getRole() {
         return role;
     }
 
-    public Address getAddress() {
+    public seedu.address.model.internship.Address getAddress() {
         return address;
     }
 
-    public Phone getPhone() {
+    public seedu.address.model.internship.Phone getPhone() {
         return phone;
     }
 
-    public Email getEmail() {
+    public seedu.address.model.internship.Email getEmail() {
```

Comment:
I believe the return value of the methods doesn't seem appropriate according to coding standard?